### DISCUSSION TITLE
GitHub Copilot chat not working in IntelliJ - always wait list

### DISCUSSION THREAD

**Discussion by manjusha-avulapati** [community/community#110859] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Thank you! You are on the waitlist for Copilot Chat in JetBrains IDEs
Admission to the private beta for GitHub Copilot Chat in JetBrains IDEs will be limited. You will receive an email if you are granted access.

**Comment by Yerock09** [comment]
Siento que me vas ah terminar de hackear pero está bien

**Comment by davecheney** [comment, marked as answer]
@manjusha-avulapati Jetbrains chat is a heavily subscribed feature. We are adding new invites as capacity allows. Thank you for your understanding.

**Comment by Aboodkaaboodkaaaa** [comment]
ما ابيه قله يكنسبه
